tools/cli/domainCommands.go

```diff
@@ -875,10 +878,19 @@ func (d *domainCLIImpl) migrationDynamicConfigCheck(c *cli.Context) DomainMigrat
 
 			if currResp.Value != newResp.Value {
 				check = false
-				mismatchedConfigs = append(mismatchedConfigs, MismatchedDynamicConfig{
-					CurrResp: currResp,
-					NewResp:  newResp,
-				})
+				mismatchedConfig = MismatchedDynamicConfig{
```

Comment:
append mismatchedConfigs here
```suggestion
				mismatchedConfigs = append(mismatchedConfigs, MismatchedDynamicConfig{/*fill in content*/})
```